roles/palava-web/files/nodesource.gpg.key

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
Does this make sense? Shouldn't we rather pull this with get_url?

Reply:
Disregard get_url. apt_key would be the right mechanism to use!

Reply:
I used the url field in the apt_key command of ansible and set it to `https://deb.nodesource.com/gpgkey/nodesource.gpg.key`